2019-11-20 13:30:11.539 - [Info] MITM Version: 1.26
2019-11-20 13:30:11.556 - [Info] Auto Access Enabled: true
2019-11-20 13:30:11.557 - [Debug] [Init] Auto Access Barrier: true
2019-11-20 13:30:11.588 - [Info] SSH man-in-the-middle server for 172.20.0.6 listening on 0.0.0.0:10105
2019-11-20 13:37:19.026 - [Debug] [Connection] Attacker connected: 43.240.125.198 | Client Identification: SSH-2.0-libssh-0.6.3
2019-11-20 13:37:20.524 - [Debug] [Auth] Attacker 43.240.125.198 trying to authenticate with "password"
2019-11-20 13:37:20.525 - [Debug] [Auto Access] Attacker: 43.240.125.198, Threshold: 1, Attempts: 1
2019-11-20 13:37:20.525 - [Debug] [Auto Access] Compromising the honeypot
2019-11-20 13:37:20.525 - [Debug] [Auto Access] Adding the following credentials: "riaz:riaz"
2019-11-20 13:37:20.812 - [Debug] [Auth] Valid credentials - Password Authentication
2019-11-20 13:37:20.812 - [Debug] [LXC] Attempting to connect to the honeypot: 172.20.0.6
2019-11-20 13:37:20.874 - [Debug] [LXC-Auth] Attacker authenticated and is inside container
2019-11-20 13:37:21.847 - [Debug] [Connection] Attacker connected: 43.240.125.198 | Client Identification: SSH-2.0-libssh-0.6.3
2019-11-20 13:37:23.330 - [Debug] [Auth] Attacker 43.240.125.198 trying to authenticate with "password"
2019-11-20 13:37:23.331 - [Debug] [Auth] Attacker authentication error: Invalid credentials - User does not exist
2019-11-20 13:37:23.332 - [Debug] [Auth] Attacker: 43.240.125.198 has so far made 1 attempts. Remaining: 5 attempts
2019-11-20 13:37:23.835 - [Debug] [Connection] Attacker closed the connection
2019-11-20 13:37:24.306 - [Debug] [EXEC] Noninteractive mode attacker command: cat /proc/cpuinfo | grep name | wc -l
2019-11-20 13:57:07.365 - [Debug] [Connection] Client error on ssh server
2019-11-20 13:57:07.367 - [Debug] [Connection] Attacker closed connection
2019-11-20 13:57:07.388 - [Debug] [LXC] Container's OpenSSH server ended connection
2019-11-20 13:57:07.389 - [Debug] [LXC] Container's OpenSSH server closed connection
